models.py

```diff
@@ -0,0 +1,28 @@
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation
+from keras.optimizers import SGD, Adam, RMSprop
```

Comment:
@mrshu Just very small nitpick... some of these imports are never used anywhere are we planing on using them later or can they be removed?